### PR TITLE
[Feature 10] Create a README.md

### LINKED ISSUE #24: [Feature 10] Create a README.md
Creating a README.md file for this project.

### PR DESCRIPTION
**Pull request by katarinaavucic**
Modified the README.md file to summarize our project.

### CODE REVIEW

**Comment by katarinaavucic** [comment]
It is a little hard to see here because of the deleted code, but this is what the README.md looks like!
Screenshot 2023-08-05 at 3 11 16 PM

Screenshot 2023-08-05 at 3 11 20 PM

**Review by YinLingithub — COMMENTED** [review]
Very nice! I think we are suppose to also include screenshots of our views, so I'll just leave some screenshots here.
Screenshot 2023-08-05 151738
Screenshot 2023-08-05 152507

**Comment by katarinaavucic** [comment]
Thank you! I was gonna ask for that but I figured it could wait til after tonight since we still have a lot to finish, but I will include them now!

File: README.md [hunk 1]
```diff
@@ -1,39 +1,98 @@
...
+| OutputData        | OD           |
+| Presenter         | P            |
+| ViewBoundary      | VB           |
+| View              | V            |
+| Gateway           | G            |
+
+
+## Clean Architecture Specifications
+
+### Clean Architecture Violations
```
Comment (YinLingithub) [inline README.md:73]: Expense and Category are basic object types we have for conveying information that are not separable. For instance, a expense called "Walmart" would have no meaning unless we know the specific amount of money spent shopping in Walmart. 
These classes belong to the Entity layer, but are passed around in both the use_case and views packages, which is a Clean Architecture violation. We could remove the imports to these entities, and change the output data to Object[][] type in the UpdateViewUCI to accommodate Java Swing requirements. However, we thought it is not adhering to SOLID principle from the point that Object[][] may not be the type required by other view libraries that could be imported, which is in contrary to the Open/Closed Principle.

**Comment by aricasas** [comment]
image

**Comment by aricasas** [comment]
image

**Review by luluwuwu — APPROVED** [review]
approve

File: README.md (hunk 1 above)
Comment (katarinaavucic) [reply README.md:73]: Thanks!

**Review by YinLingithub — APPROVED** [review]
We could take more screen shots of the GUIs when we can run the whole program on one computer.

**Review by aricasas — APPROVED** [review]
Approved

**Pull request merged**